protected/gii/CaseSearchParameter/templates/default/case_search_parameter_test.php

```diff
@@ -85,7 +84,7 @@ public function testJoin()
 
         // Ensure that the JOIN string is correct.
         //$expected = "RENDER_JOIN_HERE";
-        //$this->assertEquals($expected, $this->parameter->join('p_al_1', array('id' => 'id'), $this->searchProvider));
+        //$this->assertEquals($expected, $this->parameter->join('<?php echo $this->alias; ?>_1', array('id' => 'id'), $this->searchProvider));
```

Comment:
It is. The purpose is to streamline the process. I commented it out because it is incomplete but is useful once the code is complete.